Exiled.Installer/Program.cs

```diff
@@ -298,16 +298,14 @@ private static bool TryFindRelease(CommandSettings args, IEnumerable<Release> re
         {
             Console.WriteLine("Trying to find release..");
 
-            var range = args.TargetVersion != null ? new Range($"={args.TargetVersion}") : new Range(">2.0.0");
-
             foreach (var r in releases)
             {
                 release = r;
 
-                if (!range.IsSatisfied(r.TagName))
-                    continue;
+                if (args.TargetVersion != null && (new Version(args.TargetVersion) == new Version(r.TagName)))
```

Comment:
Wouldn't it be better to compare strings instead? Anyway, don't create a new instance for `args.TargetVersion`, it calls regex.

Reply:
Idk, it might be but I know this way works where range.IsSatisfied didn't. I'll fix the extra regex stuff though.